module/trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require './base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0, 9]
```

Comment:
- [ ] Kindly change `@nameable.correct_name[0, 9]` to `@nameable.correct_name[0, 10]` as requested by the previous reviewer as this will make the maximum length of the word 9 characters instead of 10 characters which is in this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=has%20a%20maximum%20of%2010%20characters.) because writing this way will exclude the last index.